### PR TITLE
Added pkg-config file

### PR DESCRIPTION
**Pull request by ghost**
This commit creates a pkg-config file for easy integration with build systems that utilize the program, such as Meson (see below).
It would be preferable to have an autoconf script which places this in the necessary directory but I think that is up to the maintainer.

I would appreciate this being added as it allows me (and most likely others, in the future) to easily integrate yasio in my project, without having to tell the build system where to look manually.

image

### CODE REVIEW

File: yasio.pc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+prefix=/usr/local
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${exe_prefix}/include
+
+Name: Yasio
```
Comment (halx99) [inline yasio.pc]: does 'yasio' better?

Comment (ghost) [reply yasio.pc]: done :+1:

**Comment by halx99** [comment]
looks good to me, thanks

**Comment by ghost** [comment]
thanks for really good library :) :+1:

**Pull request merged**